### PR TITLE
Release 2.1.8

### PR DESCRIPTION
**Pull request by kgaillot**

### CODE REVIEW

**Comment by kgaillot** [comment]
@clumens , can you review this? Thanks

File: ChangeLog [hunk 1]
```diff
@@ -1,54 +1,9 @@
-* Tue Jul 16 2024 Ken Gaillot <kgaillot@redhat.com> Pacemaker-2.1.8-rc4
-- 6 commits with 21 files changed, 126 insertions(+), 38 deletions(-)
-
-- Features added since Pacemaker-2.1.8-rc3
-  + tools: crm_verify now reports invalid fence topology levels
```
Comment (clumens) [inline ChangeLog:5]: I don't see this feature mentioned anywhere in the list of features since 2.1.7.

Comment (kgaillot) [reply ChangeLog:5]: oops. i also forgot a few things newer than rc4 ...

Comment (kgaillot) [reply ChangeLog:5]: updated commit

**Review by clumens — APPROVED** [review]

**Pull request merged**